### DISCUSSION TITLE
Pluto Failed to register using Julia on Windows 11, Why?

### DISCUSSION THREAD

**Discussion by itisnara** [fonsp/Pluto.jl#1906] [Questions about using Pluto] [answered]
Hello Pluto Guru,
Greetings!
My little background on using Julia:
I am a bit late comer to Julia and of course Pluto.
I was serious on doing work on Julia after hearing all good things about Julia for computational thinking as you call i.
I heard Pluto guru talk and heard introduction of MIT computational thinking Course.
I wanted to experiment using Pluto Notebook so I can create Pluto notebook for my field of expertise (including computational numerical methods and FEM modelling structures and its analyses.)
I am a retired University professor, versatile with a "BIT" of experience using FORTRAN, C, C++, Javascript, Java, Matlab, Maple, Mathematica, Octave  etc.
None of these languages came close to Julia;s plus point in terms of mathematical operators and functional programming using Julia, as I hear on YouTube.

Coming to the issue, I tried to use the PLUTO first time, using pkg add Pluto, and it failed on my HP computer with Windows 11 installed recently.  ( i heard the Pluto Guru talk on the installation of PLUTO as spoken on the Intro MIT course.)

Here are the two files: Command line snip and my HP computer system, info notepad file.

![Screenshot-of-add-Pluto-Pks-Failed- 2022-02-09 201509](https://user-images.githubusercontent.com/24774156/153321238-37497959-3640-40ee-8518-2e310ea47ba3.png)

and
[Nara-Desktop-System-Info-Feb9-2022.txt](https://github.com/fonsp/Pluto.jl/files/8037407/Nara-Desktop-System-Info-Feb9-2022.txt)

If you need more info, please do not hesitate to write to me at 'itisnara@gmail.com'

Please reply if you found why PLUTO did not get registered by Julia.

I wish to work deligently on Julia and Pluto to become a BIT more confident of using Julia/Pluto on analyzing structures.

Thanks for help
Nara
1-734-377-7470 if need to call me at your convenience

PS: I see only three section of MIT course on YOU Tube.  How to get other sections of them full 15 week course?

**Comment by icweaver** [comment, marked as answer]
Hi Nara, welcome!

Packages are case-sensitive, so could you try doing this instead?

```julia
(@v1.6) pkg> add Pluto
```

by the way, the current version of Julia out right now is v1.7, so I would definitely recommend [updating](https://julialang.org/downloads/platform/#windows) if you haven't already!

As a general rule of thumb, it helps to just ask general usage questions like this over here. I'll add a note in the duplicate issue you opened so that other folks will be directed here. For your own privacy, you may want to remove your phone number above.

The YouTube playlist for the Computational Thinking course should be [available here](https://www.youtube.com/watch?v=3zTO3LEY-cM&list=PLP8iPy9hna6T56GkMHEdSrjCCheNuEwI0), as well as through the [main course site](https://computationalthinking.mit.edu/Spring21/).

I hope that helps!

**Reply by itisnara** [reply]
Thanks Ian!
I tought I used capital 'P'.
I will try again.

BTW, how do I remove my tel number on the posted discussion page.
It does not allow me to delete tel number line now.

Thanks again.
Nara( my call name)

**Reply by itisnara** [reply]
Ian,
I did click on the two links on Computational thinking.
One starts the Introduction talk weblink.
The other on the course site that I see on youtube.
The course site has only three modules.  I do not see all 14 weeks .
Are there only three modules covering 14 weeks? I am confused.
Thanks for reply
Nara

**Reply by icweaver** [reply]
> BTW, how do I remove my tel number on the posted discussion page.
It does not allow me to delete tel number line now.

You should be able to edit your post by selecting the `Edit` option from the drop-down menu at the top right corner of your post:
![Screenshot from 2022-02-10 18-02-14](https://user-images.githubusercontent.com/25312320/153511559-11b682a1-91e4-4aff-99c0-7e9fea2f3d73.png)

It will still appear in your edit history, but it's not so big a deal. Just thought you should be aware!

> I did click on the two links on Computational thinking.
One starts the Introduction talk weblink.

Right, the playlist for the rest of the talks should be available in the sidebar. Here is a [direct link](https://www.youtube.com/playlist?list=PLP8iPy9hna6T56GkMHEdSrjCCheNuEwI0).

> The course site has only three modules. I do not see all 14 weeks . Are there only three modules covering 14 weeks?

Yes, that is my understanding. There is also the [syllabus](https://computationalthinking.mit.edu/Spring21/syllabus/) in the sidebar of the course site which includes links to each notebook and lecture video, but it appears to be a teensy bit out of sync. @fonsp should this be updated? Happy to open a PR
